### ISSUE TITLE
Field Channel images cause 500 errors (AccessControlListNotSupported)

### ISSUE THREAD

**Issue by ChristopherChudzicki** [mitodl/mit-open#219]
## Expected Behavior
Should be able to upload images (avatars, banners) to field channels at https://mit-open-rc.odl.mit.edu/admin/channels_fields/fieldchannel/

## Current Behavior
Uploading field channel images yields 500 error.

**Comment by mbertrand** [comment]
Looks like it might be an S3 bucket ACL configuration issue.

```
2023-11-15T15:54:16.209083+00:00 app[web.1]: botocore.exceptions.ClientError: An error occurred (AccessControlListNotSupported) when calling the PutObject operation: The bucket does not allow ACLs
2023-11-15T15:54:16.210404+00:00 app[web.1]: 2023-11-15 15:54:16.210 UTC [33] LOG C-0x561ac2bf66b0: db1/v-aws-app-zBefSNfaxi34H7mvCg0k-1696359839@127.0.0.1:42944 closing because: client close request (age=1s)
```

**Comment by shaidar** [comment]
This was a bit of a rabbit-hole. I was able to upload a file using the AWS CLI, however trying to do the same in Django admin was throwing the error mentioned previously. To get around it, I had to disable the newish S3 block public access feature which would revert back to relying on legacy ACL. 
The issue from [Boto docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/put_object_acl.html) is:

```If your bucket uses the bucket owner enforced setting for S3 Object Ownership, ACLs are disabled and no longer affect permissions. You must use policies to grant access to your bucket and the objects in it. Requests to set ACLs or update ACLs fail and return the AccessControlListNotSupported error code```

This is how the S3 buckets are now configured and in our code [here](https://github.com/mitodl/mit-open/blob/main/main/settings.py#L497) we're trying to set `public-read` on the object which fails since it's trying to tweak the legacy ACL which is no longer in use on the bucket. I think we can simply remove that line as the default is now private and that shouldn't impact any functionality.
